mpi-native/examples/CMakeLists.txt

```diff
@@ -0,0 +1,55 @@
+cmake_minimum_required(VERSION 3.13.0)
```

Comment:
Copied the file from `examples/CMakeLists.txt` to have the mpi examples be a separate target.